bonfire/utils.py

```diff
@@ -555,28 +551,29 @@ def hms_to_seconds(s):
 
 
 @lru_cache(maxsize=None)
-def check_hostname(hostname):
+def __check_connection(hostname, port=443, timeout=5):
```

Comment:
Being picky, but does this need the double `_`?

```suggestion
def _check_connection(hostname, port=443, timeout=5):
```

Reply:
Not picky, PEP8 specifies one leading underscore for non-public methods. Thanks for raising it